buildscripts/kokoro/gae-interop.sh

```diff
@@ -1,3 +1,34 @@
 #!/bin/sh
 
-# An empty placeholder shellscript
+set -x
+set -e
+
+pwd
+hostname
+
+gcloud info --run-diagnostics
+
+gcloud app services list
+
+
+
+# $(hostname | perl -ne '/-(\d+)$/; print $1;')
+KOKORO_GAE_APP_VERSION=$(hostname)
```

Comment:
please comment on why app_version = hostname is a good idea here (or just say we don't know and it's preliminary).